src/pages/Account/SplashPage.tsx

```diff
@@ -21,6 +21,7 @@ export const SplashPage = () => {
   const { user } = useUser();
 
   const onSubmit = () => {
+    // @debt Refactor to be using PLAYA SLUG, if needed in the future.
     history.push(user ? venueInsideUrl(PLAYA_VENUE_ID) : "/enter/step1");
```

Comment:
[OOS]
All these free literals of the type `"/enter/stepX"` should be replaced with `ENTER_STEP_1_URL`, `ENTER_STEP_2_URL`... from `urlSettings.ts`